app/services/taxonomy/publish_taxon.rb

```diff
@@ -25,6 +25,14 @@ def publish
       raise(InvalidTaxonError, I18n.t('errors.invalid_taxon'))
     end
 
+    def unpublish(type:, alternative_path: nil)
```

Comment:
I can see how it sounds a bit too much, but we don't have that many service classes at the moment. I think this pattern simplifies unit testing (by not needing `describe` and `context` blocks for example) and makes it transparent in terms of what it's expected from the class. By using a class method like `call` or similar people are not encourage to instantiate the class and play with its data, which I think is good. But it's just a suggestion :) Just brain dumping some of the discussions we had a long time ago when building `content-tagger`